src/main/java/com/ft/methodearticleinternalcomponentsmapper/messaging/MessageProducingInternalComponentsMapper.java

```diff
@@ -37,7 +35,7 @@ void mapInternalComponents(EomFile methodeContent, String transactionId, Date me
             message = messageBuilder.buildMessage(
                     internalComponentsMapper.map(methodeContent, transactionId, messageTimestamp, false)
             );
-        } catch (MethodeArticleMarkedDeletedException | MethodeArticleHasNoInternalComponentsException e) {
+        } catch (MethodeArticleMarkedDeletedException e) {
```

Comment:
Why are we no handling the other exceptions that can be thrown from the method, MethodeArticleNotEligibleForPublishException and MethodeMissingBodyException.